go.mod

```
@@ -43,6 +43,7 @@ require (
 	github.com/golang/protobuf v1.5.4
 	github.com/google/btree v1.1.3
 	github.com/google/go-cmp v0.6.0
+	github.com/google/go-github/v62 v62.0.0
```

v63 already imported below
```suggestion
	github.com/google/go-github/v62 v62.0.0
```